server/src/main/java/com/serverdemo/myserver/SearchController.java

```diff
@@ -24,6 +24,7 @@
     @GetMapping("/greeting")
     @ResponseBody
     public Greeting greeting(@RequestParam(value = "name", defaultValue = "World") String name) {
+        Iterable<Address> addresses = searchService.findAddresses(CountryCode.US);
         return new Greeting(counter.incrementAndGet(), String.format(template, name));
```

Comment:
Looks like if you copy this over to the /{countryCode} method we'll be 80% done!

Reply:
Yeah, I had it here to test/ debug since we need the full query, and regex, and validation added.

Reply:
It looks like we are already calling searchService.findAddresses(CountryCode) from the other method so I just removed the test line.